docs/advanced_reference/commands/autonomy_scaffold.md

```diff
@@ -0,0 +1,33 @@
+## Scaffold
```

Comment:
We should only have documentation from FSM scaffolding, the logic for scaffolding other components belong on open-aea